src/vochain/vochain.proto

```diff
@@ -188,6 +190,14 @@ message AdminTx {
 	bytes nonce = 11;
 }
 
+message RegisterKeyTx {
+        bytes nonce = 1;  // Unique number per vote attempt, so that replay attacks can't reuse this payload
```

Comment:
Shouldn't it be `per register attempt`?

Since there is a `weight` value, does it mean that that a single user with the same proof can register multiple keys?